iele.md

```diff
@@ -798,10 +798,11 @@ Expression calculations are simple and don't require anything but the arguments
 -   `REG = mul W0, W1` performs arbitrary-precision multiplication on W0 and W1.
 -   `REG = div W0, W1` performs arbitrary-precision t-division on W0 and W1. It is an exception to divide by zero.
 -   `REG = mod W0, W1` performs arbitrary-precision t-modulus on W0 and W1. It is an exception to modulus by zero.
+-   `REG = exp W0, W1` performs arbitrary-precision exponentiation on W0 and W1. It is an exception to exponentiate by zero.
 
 -   `REG = addmod W0, W1, W2` performs addition of W0 and W1 modulo W2. It is an exception to modulus by zero.
 -   `REG = mulmod W0, W1, W2` performs multiplication of W0 and W1 modulo W2. It is an exception to modulus by zero.
--   `REG = expmod W0, W1, W2` performs exponentiation of W0 and W1 modulo W2. It is an exception to modulus by zero.
+-   `REG = expmod W0, W1, W2` performs exponentiation of W0 and W1 modulo W2. It is an exception to exponentiate or modulus by zero.
```

Comment:
It's not at exception to exponentiate by zero. X^0 = 1.

Reply:
Yes. This was fixed in the later commit. Latest version looks like this to me:

![](https://i.imgur.com/Z1m5ipF.png)

If says: "It is an exception to exponentiate by a negative number or modulus by zero."

The commit was called: "Fixed `exp` exception description" :)

Reply:
It seems like you are watching changes for a specific commit. The latest changes are here: https://github.com/runtimeverification/iele-semantics/pull/152/files